cmake/HalideGeneratorHelpers.cmake

```
@@ -400,11 +419,12 @@ function(_Halide_add_halide_runtime RT)
     endif ()
 
     add_custom_command(OUTPUT ${GEN_OUTS}
-                       COMMAND ${ARG_FROM} -r "${TARGET}.runtime" -o . ${GEN_ARGS}
+                       COMMAND ${ARG_GENERATOR_CMD} -r "${TARGET}.runtime" -o . ${GEN_ARGS}
                        # Defers reading the list of targets for which to generate a common runtime to CMake _generation_ time.
                        # This prevents issues where a lower GCD is required by a later Halide library linking to this runtime.
                        target=$<JOIN:$<TARGET_PROPERTY:${TARGET}.runtime,Halide_RT_TARGETS>,$<COMMA>>
                        DEPENDS "${ARG_FROM}"
+                       DEPENDS "${ARG_DEPENDS}"
```

```suggestion
                       DEPENDS "${ARG_FROM}" ${ARG_DEPENDS}
```

Does having two `DEPENDS` clauses work? That makes me nervous.

---

Oops. Maybe? Willfix.